packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -352,8 +352,11 @@ export default mixins(
       }
     },
     monthClick (value: string) {
-      this.inputYear = parseInt(value.split('-')[0], 10)
-      this.inputMonth = parseInt(value.split('-')[1], 10) - 1
+      const [year, month] = value.split('-')
+
+      this.inputYear = parseInt(year, 10)
```

Comment:
what better to use `+year` instead of `parseInt()`?

Reply:
Becouse: https://stackoverflow.com/questions/17106681/parseint-vs-unary-plus-when-to-use-which/17106702.

We need integer.

Reply:
Other places use `value.split('-').map(Number)` but this is fine

Reply:
@KaelWD When to release?)